package.json

```diff
@@ -163,6 +165,9 @@
     "near-api-js": "0.44.2",
     "npm-force-resolutions": "0.0.10",
     "patch-package": "6.4.7",
-    "react-portal": "4.2.1"
+    "react-collapsed": "^3.4.0",
+    "react-portal": "4.2.1",
+    "react-select": "^5.4.0",
+    "react-tooltip": "^4.2.21"
```

Comment:
1. do we need `react-collapsed`, `react-select` & `react-tooltip` dependancies ?
2. please review lock file, i think there should not be that much changes.